draft-js/index.d.ts

```diff
@@ -1,7 +1,6 @@
 // Type definitions for Draft.js v0.9.0
 // Project: https://facebook.github.io/draft-js/
 // Definitions by: Dmitry Rogozhny <https://github.com/dmitryrogozhny>
```

Comment:
Put both authors on the same line: `// Definitions by: Dmitry Rogozhny <https://github.com/dmitryrogozhny>, Eelco Lempsink <https://github.com/eelco>`